prow/jobs/OWNERS

```diff
@@ -0,0 +1,10 @@
+filters:
+  ".*":
+    labels:
+      - prow/jobs
+
+reviewers:
+  - tooling-reviewers
+
+approvers:
+  - tooling-approvers
```

Comment:
Do we have separate OWNERS file for each subdirectory? Tooling team should not be by default owner of any subdirectory.

Reply:
tooling-team is the owner of the entire repository. This is just a precaution when one of the files that's been changed does not belong to anyone then tooling-team will decide what to do with that file. Besides every directory has its own ownership for a specific interest group regardless. This is subject of change and should be adjusted once the proposed approval flow will have a green light.

Reply:
In the future it would be best to create an on-call group that will take care of those things.

Reply:
How on call group relate to this change? What this on-call group will do?

Reply:
We can create a group of people who will be responsible for maintaining the prow jobs. Since it might not be the tooling-team only concern to take responsibility of the entire prow/jobs directory